test_semver.py

```diff
@@ -414,3 +414,42 @@ def test_parse_method_for_version_info():
     s_version = "1.2.3-alpha.1.2+build.11.e0f985a"
     v = VersionInfo.parse(s_version)
     assert str(v) == s_version
+
+
+def test_immutable():
```

Comment:
Could you also add a test to check if it's possible to add new attribute? Something like this:
```
try:
    v.some_attribute = 1
except:
    ...
```